source/openapi-spec.yml

```diff
@@ -9,13 +9,7 @@ paths:
     get:
       summary: Retrieve an application
       parameters:
-        - name: application_id
-          in: path
-          required: true
-          description: The unique ID of this application
-          schema:
-            type: string
-            maxLength: 10
+        - $ref: '#/components/parameters/application_id'
```

Comment:
Could this be `components/schemas/Application/id`? That is what it is.

(I see `components/params` is a thing, but to me the [example](https://swagger.io/docs/specification/components/) implies reusing things which only matter to the client, eg "offset", "limit" etc)

Reply:
Explained IRL. Also, I wasn't able to reference the `id` field of `Application`. 😢